extensions/amp-story-interactive/0.1/amp-story-interactive.css

```diff
@@ -17,6 +17,7 @@
 /* 1em = 16px for interactive components, unless overriden by font-size */
 .i-amphtml-story-interactive-container {
   font-family: 'Poppins', sans-serif !important;
+  --i-amphtml-story-interactive-material-easing: cubic-bezier(0.4, 0, 0.2, 1);
```

Comment:
Since this is in the context of our polls animation UX we can replace all instances of this with 
`--i-amphtml-interactive-ease-out-curve`.

Reply:
Removing the animations for now but will do!